### PR TITLE
Await response from submit endpoint

### PR DESCRIPTION
**Pull request by shannynalayna**
Waits for the response from the /submit endpoint to more accurately
display error messages that may occur.

To test: Run the gameroom example, then attempt submitting a gameroom proposal with the local node selected. (So, if you're on Acme's UI, select the 'Acme' node in the dropdown and vice-versa for bubba's UI) This should then display the error rather than a success message, as it previously did.

### CODE REVIEW

File: examples/gameroom/gameroom-app/src/scss/components/_toast.scss [hunk 1]
```diff
@@ -18,7 +18,7 @@
   display: flex;
   flex-direction: row;
   align-items: center;
-  height: 3rem;
+  height: 4rem;
```
Comment (vaporos) [inline examples/gameroom/gameroom-app/src/scss/components/_toast.scss:21]: These .scss seem unrelated to the fix

Comment (shannynalayna) [reply examples/gameroom/gameroom-app/src/scss/components/_toast.scss:21]: It's to make the toast big enough to hold the error message, otherwise it was being clipped. The scroll change, in addition, is to ensure that the error message is not too big for the toast itself.

Comment (shannynalayna) [reply examples/gameroom/gameroom-app/src/scss/components/_toast.scss:21]: I added a bit to the commit message to explain this change.

File: examples/gameroom/gameroom-app/src/store/modules/nodes.ts [hunk 2]
```diff
@@ -30,9 +30,10 @@ const getters = {
 };
 
 const actions = {
-  async listNodes({ commit }: any) {
-    const nodes = await listNodes();
-    commit('setNodes', nodes);
+  async listNodesWithoutLocal({ commit }: any, localBrand: string) {
```
Comment (agunde406) [inline examples/gameroom/gameroom-app/src/store/modules/nodes.ts:33]: Was this change supposed to be included in this commit?

Comment (shannynalayna) [reply examples/gameroom/gameroom-app/src/store/modules/nodes.ts]: Oh no, it wasn't. Think I accidentally scooped up some unrelated changes when I pushed to update the commit message. Fixed now!
